test/money_test.exs

```diff
@@ -557,6 +565,15 @@ defmodule MoneyTest do
     Application.put_env(:ex_cldr, :default_backend, cldr_backend)
   end
 
+  test "that non latn cldr are properly represented" do
+    Money.default_backend().put_locale("ar-EG")
+
+    assert Money.new(:EGP, "100")
+           |> Money.to_string() == {:ok, "‏١٠٠٫٠٠ ج.م.‏"}
```

Comment:
I don't think this test is exercising the change you've proposed since formatting doesn't pass through the parsing code. I think `"100"` would need to change to `"10,000.00"` with the appropriate localised separators.

This opens up the broader issue in that parsing still only accepts latin digits when it should accept digits in other scripts (at least digits that are non-algorithmic).

Reply:
@kipcole9 I updated the test to include a larger number so see the actual separator.

Reply:
and yes it only accepts latin digits with no separators.